distributions/Dist_mkb452.py

```diff
@@ -11,7 +11,7 @@ def __init__(self):
 	def pdf(self, x):
 		"""This is your PDF"""
 		y = (1/0.000022737)
-		return (y*np.exp(-x,3)*np.sin(x))
+		return (y*np.exp((np.power(-x,3)))*np.sin(x))
 
 	def mean(self):
 		"""This is the mean of the PDF"""
```

Comment:
looks good 
https://www.wolframalpha.com/input/?i=Sqrt%5BIntegrate%5B%28x-2.06994%29**2*%281%2F0.000022737%29*exp%28%28-x**3%29%29%29*sin%28x%29%2C+%7Bx%2C2%2C5%7D%5D%5D

Reply:
But the f_max doesn't seem right...
https://www.wolframalpha.com/input/?i=%281%2F0.000022737%29*exp%28%28-2**3%29%29%29*sin%282%29+

Reply:
it could be there are no accepts in the first tries. Maybe you can keep your code the same but change x_max to be 2.5
<img width="270" alt="image" src="https://user-images.githubusercontent.com/4458890/81028044-8e545300-8e45-11ea-8a0e-1e2cb085d8b9.png">

Reply:
FYI.. it's easiest to follow this on the webpage instead of your email:
https://github.com/cranmer/intro-exp-phys-II/pull/181

Reply:
I updated the parameters and also pushed the testing notebook. All of the codes worked after I made the updates, thank you for your help! Hopefully everything is okay now.